circus-train-avro/README.md

```diff
@@ -16,6 +16,9 @@ The avro schema file is copied into the following location if the base-url is eq
     
 Note that the `eventId` is generated by circus-train and the `.schema` folder is a hidden folder.
 
+If the specific `avro-serde-options` configuration is not provided by the user, Circus Train will still copy the external schema to the target table in a `.schema` folder.
+
+	${base-url}/eventId/.schema/<avro-file>.avsc
```

Comment:
This is the same as above right? Could take line 21 out, and merge line 19 with the above section?

Reply:
I have changed it. Does it look better now ?

Reply:
Looks great